<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-01-26T11:32:59Z" name="InetBankingV1" finished-at="2022-01-26T11:33:33Z" duration-ms="33730">
    <groups>
    </groups>
    <test started-at="2022-01-26T11:32:59Z" name="BankingApp Test" finished-at="2022-01-26T11:33:33Z" duration-ms="33730">
      <class name="com.inetbanking.testCases.TC_AddCustomerTest_003">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:com.inetbanking.testCases.TC_AddCustomerTest_003@50caa560]" started-at="2022-01-26T11:33:00Z" name="setup" finished-at="2022-01-26T11:33:11Z" duration-ms="11367" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="addNewCustomer(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:com.inetbanking.testCases.TC_AddCustomerTest_003@50caa560]" started-at="2022-01-26T11:33:13Z" name="addNewCustomer" data-provider="RegData" finished-at="2022-01-26T11:33:32Z" duration-ms="19702" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[Test123]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Male]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Bandra west]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[Mumbai]]>
              </value>
            </param>
            <param index="4">
              <value>
                <![CDATA[Maharashtra]]>
              </value>
            </param>
            <param index="5">
              <value>
                <![CDATA[1234]]>
              </value>
            </param>
            <param index="6">
              <value>
                <![CDATA[892344858]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.inetbanking.testCases.TC_AddCustomerTest_003.addNewCustomer(TC_AddCustomerTest_003.java:64)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addNewCustomer -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.inetbanking.testCases.TC_AddCustomerTest_003@50caa560]" started-at="2022-01-26T11:33:32Z" name="tearDown" finished-at="2022-01-26T11:33:33Z" duration-ms="769" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.inetbanking.testCases.TC_AddCustomerTest_003 -->
    </test> <!-- BankingApp Test -->
  </suite> <!-- InetBankingV1 -->
</testng-results>
